GithubHandlers.py

```diff
@@ -152,29 +164,54 @@ def get_file_contents(self, filepath: str, ref=None):
             traceback.print_exc()
             return "Could not retrieve file contents."
 
-    def get_pr_deltas(self, pr):
+    def get_pr_deltas(self, pr) -> str:
         """
         Gets the deltas for a pull request.
 
         Args:
             pr (PullRequest): The pull request to get deltas for.
 
         Returns:
-            str: The deltas for the pull request.
+            str: The deltas for the pull request plus additional code context.
```

Comment:
Ensure that the new path for `patch is None` in `get_file_delta` is tested in case of PRs with no file changes, returning a meaningful response.